transport/tchannel/channel_outbound.go

```diff
@@ -177,6 +183,19 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 	}, nil
 }
 
+// Introspect returns basic status about this outbound.
+func (o *ChannelOutbound) Introspect() introspection.OutboundStatus {
+	state := "Stopped"
+	if o.IsRunning() {
+		state = "Running"
+	}
+	return introspection.OutboundStatus{
+		Transport: "tchannel",
+		Endpoint:  o.addr,
+		State:     state,
```

Comment:
(same as inbound introspection)

Introspecting a TChannel transport should include the full output of https://godoc.org/github.com/uber/tchannel-go#Channel.IntrospectState.